package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cjaas/common-components",
-  "version": "2.0.0",
+  "version": "2.0.1-dev.3",
```

Comment:
Just need to commit to the minor version bump and move past the `dev` flag really

Reply:
yup, I think we decided to do 2.1.0?